CMakeLists.txt

```diff
@@ -278,6 +281,40 @@ CONFIGURE_FILE(
 	@ONLY
 )
 
+################################################################
+
+message(STATUS "
+################################################################
+Raptor Configuration Summary
+################################################################
+
+    http://librdf.org/raptor/
+
+    Configured on host $ENV{COMPUTERNAME} ${HOSTNAME}
+      host OS                   ${CMAKE_SYSTEM_NAME}
+      host architecture         ${CMAKE_SYSTEM_PROCESSOR}
+
+    General flags:
+      CC                        ${CMAKE_C_COMPILER}
+      CXX (not used)            ${CMAKE_CXX_COMPILER}
+      CFLAGS                    ${CMAKE_C_FLAGS}
+      CXXFLAGS                  ${CMAKE_CXX_FLAGS}
+      LDFLAGS                   ${CMAKE_EXE_LINKER_FLAGS}
+
+    Installation prefix:        ${CMAKE_INSTALL_PREFIX}
+
+    Dependencies (packages marked with *** are required):
+    "
+)
+
+message( "  *** Perl                        ${PERL_EXECUTABLE}")
+message( "  *** Bison                       ${BISON_EXECUTABLE}")
+message( "  *** Flex                        ${FLEX_EXECUTABLE}")
+message( "  *** libxml2 (lib)               ${LIBXML2_LIBRARIES}")
+message( "  *** libxml2 (include)           ${LIBXML2_INCLUDE_DIR}")
+
```

Comment:
looks good.  I put some of this stuff in the appveyor file too: https://github.com/dajobe/raptor/blob/master/appveyor.yml

so if we merge this, I can probably remove some